### ISSUE TITLE
[SP2019]Rules presentation (Monday morning, part of the intro)

### ISSUE THREAD

**Issue by Adimote** [sourcebots/tasks#131]
This is the intro to the game as a whole, which happens as part of our intro presentation #96.

The whole slot for that presentation is an hour, of which this should likely be about 15 minutes.

**Comment by joe9599** [comment]
@RealOrangeOne Can you do this again

**Comment by RealOrangeOne** [comment]
> Again

I've not done it before for smallpeice, but have done it twice now for SR. I've never actually seen one for smallpeice :sweat_smile:. 

I'm definitely happy to take this (and the rest of the intro presentation if there's a card for that), although I'll need some prep with someone who's done this before (cc @prophile), and someone who's had a reasonable amount of input in the rules.

**Comment by Adimote** [comment]
I had a thought that we should tell them it's up to them to get the most from the summer school, as the summer school is very free-form, and at university you shouldn't expect to be spoon-fed and still do well.

**Comment by RealOrangeOne** [comment]
This needs to be done as part of #144. I don't know whether it's worth having a separate ticket to track this, as it needs to fit in the same hour window.

**Comment by RealOrangeOne** [comment]
The game presentation currently sits in the Smallpeice 2019 folder. 

For obvious reasons, i'm not linking it here.

It's currently blocked on getting a final arena layout diagram, but it won't take long to add the images!

**Comment by RealOrangeOne** [comment]
This is now done as much as is necessary, A tiny bit of rehearsal time in the morning will cover anything which remains.
